diff --git a/texk/web2c/help.h b/texk/web2c/help.h
index 1d12856..a50cf06 100644
--- a/texk/web2c/help.h
+++ b/texk/web2c/help.h
@@ -80,6 +80,7 @@ const_string CWEAVEHELP[] = {
     "-x          omit indices and table of contents",
     "+e          enclose C material in \\PB{...}",
     "+s          print usage statistics",
+    "+r          include source locations in TeX file",
     "--help      display this help and exit",
     "--version   output version information and exit",
     NULL
